Package/Sublime Text Settings/Sublime Text Settings.sublime-syntax

```diff
@@ -5,37 +5,68 @@ file_extensions:
   - sublime-settings
 scope: source.json.sublime.settings
 contexts:
-  main:
+  prototype:
     - include: JSON.sublime-syntax#comments
+
+  main:
     - match: '\{'
-      scope: punctuation.section.dictionary.begin.json
+      scope: punctuation.section.mapping.begin.json
       push:
-        - meta_scope: meta.structure.dictionary.json
+        - meta_scope: meta.mapping.json
         - include: JSON.sublime-syntax#comments
         - match: '\}'
-          scope: punctuation.section.dictionary.end.json
+          scope: punctuation.section.mapping.end.json
           pop: true
         - match: '"'
           scope: punctuation.definition.string.begin.json
-          push:
-            - meta_scope: meta.structure.dictionary.key.json.sublime.settings string.quoted.double.json
-            - meta_content_scope: keyword.other.name.key.sublime.settings
-            - match: '"'
-              scope: punctuation.definition.string.end.json
-              pop: true
-            - match: $\n?
-              scope: invalid.illegal.unclosed-string.json
-              pop: true
-        - match: ":"
-          scope: punctuation.separator.dictionary.key-value.json
-          push:
-            - meta_scope: meta.structure.dictionary.value.json
-            - match: '(,)|(?=\})'
-              captures:
-                1: punctuation.separator.dictionary.pair.json
-              pop: true
-            - include: JSON.sublime-syntax#value
-            - match: '[^\s,]'
-              scope: invalid.illegal.expected-dictionary-separator.json
-        - match: '[^\s\}]'
-          scope: invalid.illegal.expected-dictionary-separator.json
+          push: [post-key, inside-key]
+        - match: \S
+          scope: invalid.illegal.expected-key.json
+    - match: \S
+      scope: invalid.illegal.expected-mapping.sublime-settings
+
+  inside-key:
+    - clear_scopes: 1
+    - meta_scope: meta.mapping.key.json string.quoted.double.json
+    - meta_content_scope: entity.name.other.key.sublime-settings
```

Comment:
From my point of view we should either use `sublime.settings` or `sublime-settings` everywhere. But at the moment you use first expression on line 6 for the global source scope and second one here for keywords.

This is some kind of inconsistent.